### DISCUSSION TITLE
Is It Appropriate To Submit A Bounty PR For An Idea I Intend To Claim Myself?

### DISCUSSION THREAD

**Discussion by jsardello** [XRPLBounties/Proposals#83] [Q&A] [answered]
I'm interested in creating an XRPL plugin for the Godot game engine. Since there are already 2 game engine bounties for Unity and Unreal, I'm assuming that Bounties would be a better fit rather than the Grants program. I'm unsure if it's taboo to submit for a bounty idea where I intend to pursue the bounty myself. Feedback welcome.

**Comment by JST5000** [comment, marked as answer]
Hey @jsardello, thanks for sharing this idea!

I think it would probably be best to submit it for a [grant](https://xrplgrants.org/) because it’s work that you want to do. That will let you define what budget you feel is reasonable and define the scope of the work you want to do. Creating a bounty for work you want to pick up yourself is a bit counter to our goal of using bounties to encourage new XRPL developers to pick up meaningful work.

From what I’ve read up on Godot it certainly seems interesting, and like the developer community is very active, so it seems like a cool idea for a grant :slightly_smiling_face:

**Reply by jsardello** [reply]
Great, thanks for the clarification @JST5000. I'll start molding my ideas into a grant proposal.
